tests/TestUrlTools.cpp

```diff
@@ -136,6 +136,22 @@ void TestUrlTools::testIsUrlValid()
     }
 }
 
+void TestUrlTools::testIsUrlValidWithLooseComparison()
+{
+    QHash<QString, bool> urls;
+    urls["\"https://github.com/login\""] = true;
+    urls["https://*.github.com/"] = true;
+    urls["*.github.com"] = true;
+    urls["*.com"] = true;
+    urls["https://*.com"] = true;
```

Comment:
Need to add a test for a false condition, some examples:

Exact urls (in quotes) with wildcards:
`"*.example.com"`
`""`

When these are not valid:
`*`
`*.com`